Base/usr/share/man/man1/arp.md

```diff
@@ -0,0 +1,21 @@
+## Name
+
+arp - find near devices in the local network
```

Comment:
Technically this program doesn't find devices on the local network. It prints the ARP cache but does not actually go out of its way to find devices.